src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
I guess what I am pointing out is that there are semantics implied in `{ type: 'loaded:foo' }` that are missing from `{ type: Symbol('foo') }`.  With one, you are being explicit that the event is a loaded event, the other, it is simply implied because that is the only event emitted.  I can understand that it works, but I am just pointing out there is an inconsistency.  Furthering that logic if `{ type: 'foo' }` wouldn't make sense, why does `{ type: Symbol('foo') }` make sense?  You _could_ in the future have some other event type for a string label, but you _cannot_ have another event type for a symbol label.